### PR TITLE
Added support for Azure SQL

### PR DESCRIPTION
**Pull request by Andrewpk**
[sys].[extended_properties] isn't available on Azure SQL so generating anything from those comments is not possible.

### CODE REVIEW

**Comment by Andrewpk** [comment]
As a note, without this gii and activerecord will typically fail with `yii\base\InvalidConfigException` saying `The table does not exist: TABLENAME` any time `yii\db\ActiveRecord::getTableSchema()` gets called.

**Comment by klimov-paul** [comment]
It whould be better if separated `AzureSchema` class is created instead of using boolean field.

**Comment by samdark** [comment]
Agree.

**Comment by Andrewpk** [comment]
@samdark @klimov-paul Sounds good. I was unsure if you'd want these changes as an additional schema initially so I just added them to MSSQL. I can make those changes as soon as I'm not at work tonight if you'd like.

**Comment by Andrewpk** [comment]
@samdark @klimov-paul The changes you requested were made almost a year ago. Are there any other issues that are keeping this from being accepted?

**Comment by samdark** [comment]
Could you please merge with master?

File: framework/db/mssql/AzureSchema.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * @author Andrew Kehrig <me@andrewkehrig.com>
+ */
+class AzureSchema extends yii\db\mssql\Schema {
+    
+    /**
+     * 
+     * @param array $info column information
+     * @return string the comment property for ColumnSchema column object
+     */
+    protected function getLoadColumnSchemaComment($info)
```
Comment (samdark) [inline framework/db/mssql/AzureSchema.php]: Is it better to name it `getColumnComment()`?

File: framework/db/mssql/AzureSchema.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+/**
+ * Simple Schema class for accessing Microsoft Azure hosted databases.
+ *
+ * @author Andrew Kehrig <me@andrewkehrig.com>
+ */
+class AzureSchema extends yii\db\mssql\Schema {
+    
+    /**
+     *
```
Comment (samdark) [inline framework/db/mssql/AzureSchema.php]: Method description is required.

File: framework/db/mssql/AzureSchema.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+     * @param array $info column information
+     * @return string the comment property for ColumnSchema column object
+     */
+    protected function getLoadColumnSchemaComment($info)
+    {
+        return '';
+    }
+    
+    /**
+     *
```
Comment (samdark) [inline framework/db/mssql/AzureSchema.php]: Method description is required.

File: framework/db/mssql/Schema.php [hunk 4]
```diff
@@ -222,6 +222,16 @@ protected function loadColumnSchema($info)
 
         return $column;
     }
+    
+    /**
+     *
```
Comment (samdark) [inline framework/db/mssql/Schema.php]: Method description is required.

File: framework/db/mssql/Schema.php [hunk 5]
```diff
@@ -280,6 +276,30 @@ protected function findColumns($table)
 
         return true;
     }
+    
+    /**
+     *
```
Comment (samdark) [inline framework/db/mssql/Schema.php]: Method description is required.

**Comment by cebe** [comment]
This PR seems outdated, closing it for now. @Andrewpk are you willing to work on this further? If so let us know and we will reopen it. Thanks for your contribution!

**Pull request closed without merging**